src/code/fault.c

```diff
@@ -184,7 +182,38 @@ void Fault_RemoveClient(FaultClient* client) {
     }
 }
 
+#ifdef NON_MATCHING
+// minor ordering differences
+void Fault_AddAddrConvClient(FaultAddrConvClient* client, void* callback, void* param) {
+    FaultAddrConvClient* iter;
+    bool alreadyExists = false;
+    OSIntMask mask;
+
+    mask = osSetIntMask(1);
+
+    iter = sFaultStructPtr->addrConvClients;
+    while (iter) {
```

Comment:
Generally we only compare boolean values as booleans. 
```suggestion
    while (iter != NULL) {
```